drivers/usb_c/tcpc/ucpd_stm32.c

```diff
@@ -149,6 +149,28 @@ static void stm32_ucpd_state_init(const struct device *dev)
 	data->ucpd_vconn_enable = false;
 }
 
+/**
+ * @brief Get the CC enable mask. The mask indicates which CC line
+ *        is enabled.
+ *
+ * @retval CC Enable mask (bit 0: CC1, bit 1: CC2)
+ */
+static uint32_t ucpd_get_cc_enable_mask(const struct device *dev)
+{
+	struct tcpc_data *data = dev->data;
+	const struct tcpc_config *const config = dev->config;
+	uint32_t mask = UCPD_CR_CCENABLE_Msk;
+
+	if (data->ucpd_vconn_enable) {
```

Comment:
Add a comment indicating that supplying VCONN disables the corresponding CC line.

Reply:
done